modules/javafx.media/src/main/java/com/sun/media/jfxmediaimpl/NativeMediaManager.java

```diff
@@ -268,7 +289,6 @@ public static MetadataParser getMetadataParser(Locator locator) {
     public MediaPlayer getPlayer(Locator locator) {
         // FIXME: remove this
         initNativeLayer();
-
```

Comment:
Maybe best to revert this change (i.e., restore the blank line) lest someone think that the FIXME comment applies to the creation of the player?